src/main/java/hudson/plugins/claim/ClaimPublisher.java

```diff
@@ -63,11 +74,10 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
             }
         }
 
-        return true;
     }
 
     public BuildStepMonitor getRequiredMonitorService() {
-        return BuildStepMonitor.STEP;
+        return BuildStepMonitor.NONE;
     }
```

Comment:
From what I can tell, there shouldn't be an impact - claim is just adding an action, not looking at history.